src/protocol/payload/version.rs

```diff
@@ -51,7 +51,8 @@ impl Version {
                 addr: addr_from,
             },
             nonce: Nonce::default(),
-            user_agent: VarStr(String::from("")),
+            // Let's pretend to be a ZCashd node 5.4.2
+            user_agent: VarStr(String::from("MagicBean:5.4.2")),
```

Comment:
Let's make this a constant instead

Reply:
Done

But usually, I follow a rule: if a magic value appears only once, creating a const is not necessary, it's just an overhead